src/Compilers/Core/Portable/CommandLine/AnalyzerConfigSet.cs

```diff
@@ -199,10 +187,47 @@ public AnalyzerConfigOptionsResult GetOptionsForSourcePath(string sourcePath)
             // exact same options
             if (!_optionsCache.TryGetValue(sectionKey, out var result))
             {
+                var treeOptionsBuilder = _treeOptionsPool.Allocate();
+                var analyzerOptionsBuilder = _analyzerOptionsPool.Allocate();
+                var diagnosticBuilder = ArrayBuilder<Diagnostic>.GetInstance();
+
+                int sectionIndex = 0;
+                for (int analyzerConfigIndex = 0;
+                    analyzerConfigIndex < _analyzerConfigs.Length && sectionIndex < sectionKey.Count;
```

Comment:
Nit: formatting (1 space off)

Reply:
This is the correct indentation (indent by one tab, which is 4 spaces).

Reply:
It certainly doesn't look correct, as it's not lined up with the previous line.

Reply:
Agreed.  It's a very common idiom to align all the children of a for-loop when wrapping.

Reply:
Agreed, this should align.

Reply:
Filed https://github.com/dotnet/roslyn/issues/43119